docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       - API_ADDRESS=0.0.0.0         # Address of this machine
       - API_PORT=4445               # Port you wish to use for API
       - API_KEY=                    # Auth key you wish to set
-      - OBS_URL=ws://127.0.0.1:4444 # WebSocket Connect URL
+      - OBS_URL=ws://127.0.0.1:4455 # WebSocket Connect URL
       - OBS_PASSWORD=               # OBS password, if used
     ports:
       - "4445:4445"             # Set to same value as API_PORT
```

Comment:
Sticking with the trend of ws + 1, let's use port 4456 by default

Reply:
This should be 4455 because it is set that way by default in OBS versions => 28

Reply:
I was referring to the usage of port 4445 as API_PORT